Examples/Scripts/CorrThickPipelineBatch.sh

```diff
@@ -0,0 +1,91 @@
+get_batch_options() {
```

Comment:
To make it clear this isn't a tcsh script.
```suggestion
#!/bin/bash

get_batch_options() {
```